json-endpoints/jetpack/class.jetpack-json-api-themes-modify-endpoint.php

```diff
@@ -92,34 +92,32 @@ function update_translations() {
 		$available_themes_updates = get_site_transient( 'update_themes' );
 		
 		if ( ! isset( $available_themes_updates->translations ) || empty( $available_themes_updates->translations ) ) {
-			
 			return new WP_Error( 'nothing_to_translate' );
 		}
 
 		foreach( $available_themes_updates->translations as $translation ) {
+			$theme = $translation['slug'] ;
 			if ( ! in_array( $translation['slug'], $this->themes )  ) {
+				$this->log[ $theme ][] = __( 'No update needed', 'jetpack' );
 				continue;
 			}
 
-			$theme = $translation['slug'] ;
-
-
 			/**
 			 * Pre-upgrade action
 			 *
-			 * @since 3.9.3
+			 * @since 4.4
 			 *
 			 * @param object $theme WP_Theme object
 			 * @param array $themes Array of theme objects
 			 */
-			do_action('jetpack_pre_theme_upgrade', $theme, $this->themes);
```

Comment:
Since you deprecated that hook, you'll need to add it to the list of deprecated hooks in `class.jetpack.php`